website/documentation/guides/monitoring-and-troubleshooting/shoot-deletion-stuck-due-to-unmanaged-resources.md

```diff
@@ -0,0 +1,226 @@
+```
+
+Replace `< subnet-id >` with the ID of the subnet you want to inspect.
+
+##### 5. List Ports
+To find out if there are any ports still attached to the network, use:
+```
+openstack port list --network <network-id>
+```
+This command will list all ports associated with the network, which might prevent its deletion.
+
+##### 6. Show Port Details
+To get detailed information about a specific port, use:
+```
+openstack port show <port-id>
+```
+
+Replace `< port-id >` with the ID of the port you want to inspect.
+
+##### 7. List Routers
+To check if there are any routers connected to the network, use:
+```
+openstack router list
+```
+This will list all routers in your environment. You can then check if any are connected to the network in question.
+
+##### 8. Show Router Details
+To get detailed information about a specific router, use:
+```
+openstack router show <router-id>
+```
+Replace `< router-id >` with the ID of the router you want to inspect.
+
+##### 9. Check for Floating IPs
+To see if there are any floating IPs associated with the network, use:
+```
+openstack floating ip list
+```
+This will list all floating IPs, which might be associated with resources in the network.
+
+##### 10. Delete Resources
+Once you've identified any resources that might be preventing the network deletion, use the appropriate openstack commands to delete them, such as:
+```
+openstack port delete <port-id>
+openstack subnet delete <subnet-id>
+openstack router delete <router-id>
+```
+
+#### Case 3: Shoot stuck due to AWS LoadBalancers
+
+Your shoot reports message of type:
+
+```
+Error deleting Infrastructure: 2 errors occurred:\n\t* failed to \"Destroying Kubernetes load balancers and security groups\": failed to destroy load balancers and security groups: could not delete loadbalancer for arn arn:aws:elasticloadbalancing:eu-central-1:xxxxxxx:loadbalancer/net/some-load-balancer-name/d22d72f6f5563eff: operation error Elastic Load Balancing v2: DeleteLoadBalancer, https response error StatusCode: 400, RequestID: 9e0ad839-dc0d-4809-a6d6-1d8373ad8516, ResourceInUse: Load balancer 'arn:aws:elasticloadbalancing:eu-central-1:xxxxxxxxx:loadbalancer/net/some-load-balancer-name/d22d72f6f5563eff' cannot be deleted because it is currently associated with another service\n\t* failed to \"delete zones resources\": 2 errors occurred:\n\t* failed to \"delete subnet resource eu-central-1a-SubnetPublicUtility\": operation error EC2: DeleteSubnet, context deadline exceeded\n\t* failed to \"delete subnet resource eu-central-1b-SubnetPublicUtility\": operation error EC2: DeleteSubnet, context deadline exceeded\n\n\n\n"
+```
+
+##### 1. Load Balancer Deletion Issue
+- **Identify Associations**: The load balancer is associated with another service, which is preventing its deletion. Use the AWS Management Console or CLI to identify what services or resources are using the load balancer. This could include target groups, listeners, or other AWS services.
```

Comment:
```suggestion
- **Identify Associations**: The load balancer is associated with another service, which is preventing its deletion. Use the AWS Management Console or CLI to identify what services or resources are using the load balancer. This could include target groups, listeners or other AWS services.
```